ros_buildfarm/templates/ci/ci_create_tasks.Dockerfile.em

```diff
@@ -59,7 +59,7 @@ args = \
     ' --workspace-root ' + ' '.join(workspace_mount_point) + \
     ' --distribution-repository-urls ' + ' '.join(distribution_repository_urls) + \
     ' --distribution-repository-key-files ' + ' ' .join(['/tmp/keys/%d.key' % i for i in range(len(distribution_repository_keys))]) + \
-    ' --env-vars ' + ' ' .join(env_vars)
+    ' --env-vars ' + ' ' .join(['%s=%s' % var for var in env_vars.items()])
```

Comment:
Nitpick: while this works it takes a lot of re-reading to figure out that `var` actually is a tuple with the name and the value. At least use a more intuitive variable name like `key_value` or use two variables.

Reply:
Addressed in 3fb42fb